doc/api/async_hooks.md

```diff
@@ -973,6 +973,21 @@ chains. It allows storing data throughout the lifetime of a web request
 or any other asynchronous duration. It is similar to thread-local storage
 in other languages.
 
+## Difference with raw Async Hooks
+
+`AsyncLocalStorage` doesn't provide any new functionality that raw Async Hooks
```

Comment:
In fact, ALS does provide new functionality when compared to "raw" `async_hooks`. It's more strict to say that it builds on top of other features available in `async_hooks` module.